messages/sv/base.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
```

Comment:
what is this sv/base file about?

Reply:
It's a translation file that belongs the last PR, i forgot to add it when pushing the code.